### PR TITLE
ohdear-cli 4.0.0

### PR DESCRIPTION
**Pull request by owenvoke**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

I'm not entirely sure how changes to the binary name are dealt with in Brew 🤔

### CODE REVIEW

File: Formula/ohdear-cli.rb [hunk 1]
```diff
@@ -12,10 +12,10 @@ class OhdearCli < Formula
   depends_on "php"
 
   def install
-    bin.install "ohdear-cli.phar" => "ohdear-cli"
+    bin.install "ohdear.phar" => "ohdear"
```
Comment (chenrui333) [inline Formula/ohdear-cli.rb:15]: maybe add a symlink for `ohdear-cli`?

Comment (owenvoke) [reply Formula/ohdear-cli.rb:15]: Is there a good way to handle that in Brew, or just run `ln -s`? 🤷🏻

Comment (chenrui333) [reply Formula/ohdear-cli.rb:15]: check out `bin.install_symlink` in the repo.

Comment (chenrui333) [reply Formula/ohdear-cli.rb:15]: Also does upstream have preference one over another?

Comment (owenvoke) [reply Formula/ohdear-cli.rb:15]: The preference is to use `ohdear`, as that's the new name going forward. 👍🏻

Comment (chenrui333) [reply Formula/ohdear-cli.rb:15]: Thanks for those comments, that is pretty helpful. Also can you squash the commits? I think we are good to go after it :)

**Review by p-linnane — APPROVED** [review]

**Pull request merged**